flask_cors/core.py

```diff
@@ -9,7 +9,12 @@
 """
 import re
 import logging
-import collections
+try:
+    # on python 3
+    from collections.abc import Iterable
+except ImportError:
```

Comment:
How about moving this a couple of lines below and using `six.PY` after it is imported?

Something like:
```python
from sis import string_types, PY3
if PY3:
    from collections.abc import Iterable
else:
    from collections import Iterable
```

Reply:
Thanks for reviewing. It seems to be equivalent but using six instead of traditional ImportError approach. When I have 5 minutes available will test your recommendation but I'm pretty confident it will do the same job.

Reply:
I’m ambivalent about the two!